apps/dash-mapd-demo/README.md

```diff
@@ -0,0 +1,63 @@
+# Dash-mapd-demo
+
+## About this app
+
+A sample dash app which updates real-time visual analytics by constantly queries from a remote Omnisci(MapD) SQL Engine. 
+
+## About this dataset
+
+The public dataset using in this app, "flight_2008_7M" includes every single record of US airline flights, known by Bureau of Transportation Statistics in the year 2008. By default, it could be loaded from `insert_sample_data` script during Omnisci server installation. A [jupyter notebook](https://github.com/plotly/dash-mapD-demo/blob/dev/flights_data_clean.ipynb) has been included to showcase tabular structure and query results from this dataset.
```

Comment:
```suggestion
The public dataset used in this app, "flight_2008_7M", includes every single record of US airline flights known by  the Bureau of Transportation Statistics in the year 2008. By default, it can be loaded from the `insert_sample_data` script during the Omnisci server installation. A [jupyter notebook](https://github.com/plotly/dash-mapD-demo/blob/dev/flights_data_clean.ipynb) has been included to showcase tabular structure and query results from this dataset.
```